ComponentKit/Core/CKComponentViewConfiguration.mm

```diff
@@ -20,7 +20,9 @@
 
 CKComponentViewClass::CKComponentViewClass(Class viewClass) noexcept :
 identifier(class_getName(viewClass)),
-factory(^{return [[viewClass alloc] init];}) {}
+factory(^{ return [[viewClass alloc] init]; }) {
+  CKAssertTrue([viewClass isSubclassOfClass:[UIView class]], @"%@ is not a subclass of UIView", viewClass);
```

Comment:
You probably wanted to use CKAssert here - CKAssertTrue doesn't take description